app/menus/DocumentMenu.js

```diff
@@ -225,6 +230,11 @@ class DocumentMenu extends React.Component<Props> {
               New nested document
             </DropdownMenuItem>
           )}
+          {!document.isArchived && !document.isDeleted && !document.isDraft && (
```

Comment:
```suggestion
          {can.unpublish && (
```

`can.unpublish` should be available here as you defined it as a policy.